File: jeonghyeon/4week/나머지가 1이 되는 수 찾기.js
function solution(n) {
    for(let i = 1; i < n; i++) { // 1부터 n까지의 수를 늘려가며 찾는다
        if(n % i == 1) { // n이 i로 나눠져서 나머지가 1이 되면 i 리턴
            return i;
        }
    }
}

File: minwook/4주차/나머지가_1이_되는_수_찾기.java
class Solution {
    public int solution(int n) {
        int answer = 0;
        for(int i=1; i<n; i++){
            if(n%i == 1){ // 나머지가 1이면 종료
                answer = i;
                break;
            }
        }
        return answer;
    }
}